### ISSUE TITLE
How does it compare to obs-tablet-remote?

### ISSUE THREAD

**Issue by fbartels** [Niek/obs-web#6]
Hi,

I've learned about your project through the "Show HN". So far I am using http://t2t2.github.io/obs-tablet-remote/.

What would you say are the strengths of your project compared to obs-tablet-remote?

And where would you say is it lacking?

I've noticed that you brought up live preview in the obs-tablet-remote bugtracker as well https://github.com/t2t2/obs-tablet-remote/issues/73

**Comment by Niek** [comment]
Good to hear!

This project is similar to obs-tablet-remote, I wrote OBS-web because I was missing some features in obs-tablet-remote like a preview of the output and the ability to hide scenes. I looked into adding them, but Vue is not my strongest point (I like Svelte a lot) - hence this project. Another feature of OBS-web is that the password requirement is detected upon login, so it makes it a little bit easier to use. But honestly obs-tablet-remote is a great app as well and allows more customizations - so use whatever you prefer :)
